util/crc32c_ppc.c

```diff
@@ -0,0 +1,101 @@
+/* Copyright (C) 2017 International Business Machines Corp.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
```

Comment:
@sagar0 is going to ask our lawyer for that. In the mean time, I think these files should go under third-party/.

@grooverdan what do you mean by "you can merge this under any open source license you choose". Do you mean you can change this header if needed according to Facebook's lawyer?